res/css/views/rooms/_EventTile.pcss

```diff
@@ -584,6 +578,14 @@ $left-gutter: 64px;
                 padding-inline-start: calc(var(--EventTile_group_line-spacing-inline-start) + 20px);
             }
         }
+
+        &.mx_EventTile_highlight,
+        &.mx_EventTile_highlight .markdown-body {
+            > .mx_EventTile_line {
+                box-shadow: inset var(--EventTile-box-shadow-offset-x) 0 0 var(--EventTile-box-shadow-spread-radius)
+                    $event-highlight-edge-color;
```

Comment:
Any new styles should be using CPD tokens exclusively https://compound.element.io/?path=/docs/foundations-design-tokens--docs

Reply:
I recognize now that for this, I would need to open a separate companion PR in https://github.com/element-hq/compound-design-tokens to add the semantic tokens necessary for this change.

— In the interest of moving forward, I'm dropping this addition from the PR for now (the highlighted left-edge on mentions) and plan to circle back to it in a future PR.